tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/contact/EmailAddressContactSearchEngineContract.scala

```diff
@@ -52,6 +54,28 @@ trait EmailAddressContactSearchEngineContract {
 
   def awaitDocumentsIndexed(query: QueryType, documentCount: Long): Unit
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("d", "di", "dia", "dian", "diana"))
```

Comment:
"d" should be removed now that the minimum is 2?

Reply:
Yes, good catch